app/src/screens/PersonCredential.tsx

```diff
@@ -89,45 +56,66 @@ const PersonCredential: React.FC = () => {
     )
   }, [])
 
-  return (
-    <SafeAreaView style={styles.pageContainer}>
-      <ScrollView contentContainerStyle={[styles.pageContent]}>
-        <View>
-          <View>
-            <Text style={TextTheme.normal}>
-              {t('PersonCredential.Description') + ' '}
-              <TouchableOpacity onPress={getBCServicesCardApp}>
-                <Text style={{ ...TextTheme.normal, color: ColorPallet.brand.link }}>
-                  {t('PersonCredential.LinkDescription')}
-                </Text>
-              </TouchableOpacity>
-            </Text>
-          </View>
+  const personCredentialAttributes = {
+    credName: 'Person',
+    credDefId: 'RGjWbW1eycP7FrMf4QJvX8:3:CL:13:Person',
+    schemaId: 'XUxBrVSALWHLeycAUhrNr9:2:Person:1.0',
+    attributes: [{ name: 'given_names', value: t('PersonCredential.GivenName') }, { name: 'family_name', value: t('PersonCredential.FamilyName') }]
+  }
+
+  const personPageFooter = () => {
+    return (
+      <View>
+        <View style={styles.button}>
+          <Button
+            title={t('PersonCredential.GetCredential')}
+            accessibilityLabel={t('PersonCredential.GetCredential')}
+            onPress={startGetBCIDCredentialWorkflow}
+            disabled={workflowInFlight}
+            buttonType={ButtonType.Primary}
+          >
+            {workflowInFlight && (
+              <LoadingIcon color={ColorPallet.grayscale.white} size={35} active={workflowInFlight} />
+            )}
+          </Button>
         </View>
-        <View style={styles.buttonContainer}>
-          <View style={styles.button}>
-            <Button
-              title={t('PersonCredential.GetCredential')}
-              accessibilityLabel={t('PersonCredential.GetCredential')}
-              onPress={startGetBCIDCredentialWorkflow}
-              disabled={workflowInFlight}
-              buttonType={ButtonType.Primary}
-            >
-              {workflowInFlight && (
-                <LoadingIcon color={ColorPallet.grayscale.white} size={35} active={workflowInFlight} />
-              )}
-            </Button>
-          </View>
-          <View style={styles.button}>
-            <Button
-              title={t('PersonCredential.Decline')}
-              accessibilityLabel={t('PersonCredential.Decline')}
-              onPress={dismissPersonCredentialOffer}
-              buttonType={ButtonType.Secondary}
-            ></Button>
-          </View>
+        <View style={styles.button}>
+          <Button
+            title={t('PersonCredential.Decline')}
+            accessibilityLabel={t('PersonCredential.Decline')}
+            onPress={dismissPersonCredentialOffer}
+            buttonType={ButtonType.Secondary}
+          ></Button>
         </View>
-      </ScrollView>
+      </View>
+    )
+  }
+  return (
+    <SafeAreaView style={styles.pageContainer} edges={['bottom', 'left', 'right']}>
```

Comment:
The idea was to reuse the CardComponent so we don't have to update this every time the credential branding changes

Reply:
we are reusing the card component, see line 105. All the other stuff is for the wording on the page and the flatlist is so that the page content will be scrollable

Reply:
Lol 🤦‍♂️. I scrolled through that too fast